src/sxapi/base.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python
+# coding: utf8
+
+import requests
+
+PUBLIC_API_V2 = "https://api.smaxtec.com/api/v2"
```

Comment:
these should be used somewhere...

Reply:
will be, I just wanted them to be specified for now!

Reply:
I'd suggest implementing two classes that derive from `BaseAPI` that use these URLs.